README.md

```diff
@@ -13,3 +13,31 @@ Contents of this repo:
 ## License
 
 (c) Google, 2015. Licensed under [Apache-2](LICENSE) license.
+
+## Usage
+
+First, create an unencrypted private key for use with account registration. This can be done using OpenSSL:
```

Comment:
Sure thing. It should work with `goacme reg -gen mailto:x@example.com`.
I had some reasoning behind `-gen` being false by default but happy to hear alternatives. Maybe `-gen` could be true by default.

```
$ goacme help reg
usage: goacme reg [-c config] [-gen] [-d url] [contact [contact ...]]

Reg creates a new account at a CA using the discovery URL
specified with -d argument. The default value is letsencrypt.
For more information about the discovery run goacme help disco.

Upon successful registration, a new config will be written to account.json
in the directory specified with -c argument. Default location of the config dir
is /home/crhyme/.config/acme.
If the config dir does not exist, it will be created.

Contact arguments can be anything: email, phone number, etc.

If -gen flag is not specified, and an account key does not exist, the command
will exit with an error.
```